clang-tools-extra/clangd/Protocol.h

```diff
@@ -1451,6 +1452,19 @@ struct RenameParams {
 bool fromJSON(const llvm::json::Value &, RenameParams &, llvm::json::Path);
 llvm::json::Value toJSON(const RenameParams &);
 
+struct SearchASTArgs {
```

Comment:
Please call this `SearchASTParams` for consistency with other names in this file